server/src/main/java/org/opensearch/rest/action/cat/RestIndicesAction.java

```diff
@@ -694,8 +694,8 @@ protected Table getTableWithHeader(final RestRequest request) {
         table.addCell("suggest.total", "sibling:pri;alias:suto,suggestTotal;default:false;text-align:right;desc:number of suggest ops");
         table.addCell("pri.suggest.total", "default:false;text-align:right;desc:number of suggest ops");
 
-        table.addCell("memory.total", "sibling:pri;alias:tm,memoryTotal;default:false;text-align:right;desc:total used memory");
-        table.addCell("pri.memory.total", "default:false;text-align:right;desc:total user memory");
+        table.addCell("memory.total", "sibling:pri;alias:tm,memoryTotal;default:false;text-align:right;desc:total used memory of primaries & replicas");
```

Comment:
```suggestion
        table.addCell("memory.total", "sibling:pri;alias:tm,memoryTotal;default:false;text-align:right;desc:total used memory (primary + replicas)");
```

to keep it a bit shorter may be

Reply:
Looking at above help options, this change will make it inconsistent with the other help options.

For example:
```
table.addCell("suggest.time", "sibling:pri;alias:suti,suggestTime;default:false;text-align:right;desc:time spend in suggest");
table.addCell("pri.suggest.time", "default:false;text-align:right;desc:time spend in suggest");
```


**I propose we only fix the type `user` to `used`**, else to keep consistency we should ideally change it for places which use `pri.` prefix which I don't see helpful. We already have a help option at the start of table for `pri` & `rep` which would make things explanatory.